fiftyone/server/aggregate.py

```diff
@@ -117,7 +117,6 @@ class AggregateQuery:
     @gql.field
     async def aggregate(
         self,
-        *,
```

Comment:
@kaixi-wang I removed a number of `*` args in method calls that were introduced by the saved view work. I don't see any reason why they are actually needed?

Was it because certain positional arguments to functions were renamed and you wanted to avoid an error being raised if a  `dataset_name` positional arg used to be called `name`?

I always try to enforce a strict convention that `key=value` syntax is never used when the function arg is positional, not keyword. So it should be allowed to rename positional args.

I'd like to avoid adding `*` in places where it isn't needed

Reply:
I added them because it was hard to infer/recall what needed to be passed to the functions in what order. Since there aren't type annotations, more than a couple args, and overall many similarly named functions with slightly different signatures, it felt really cumbersome/precarious

Reply:
The `*` just forces callers to use `key=value` syntax, which is already something that all code we write should do for optional function parameters.

Reply:
This seems like more of a stylistic thing that we should either intentionally adopt everywhere, or nowhere.